spec/spec_helper.rb

```diff
@@ -36,8 +36,10 @@ def json_response(*path, endpoint)
 def request_headers
   {
     "Accept" => "application/json",
+    "Accept-Encoding" => "gzip;q=1.0,deflate;q=0.6,identity;q=0.3",
```

Comment:
Could you provide some context on this change? Is this necessary for the test suite?